packages/lab/stories/Badge.stories.newapp-badge.css

```diff
@@ -0,0 +1,8 @@
+.uitk-success.uitk-newapp {
```

Comment:
we should provide these as a className to the badge ("uitkBadge-success" on the component) and then have .uitk-newapp .uitkBadge-success {  
  --uitkBadge-content-icon-color: var(--uitk-status-success-foreground);
  --uitkBadge-content-text-color: var(--uitk-color-white);
}
and same for error! :)